libcxx/test/std/containers/views/mdspan/extents/ctad.pass.cpp

```diff
@@ -35,6 +41,7 @@ constexpr bool test() {
   test(std::extents(1, 2u), std::extents<std::size_t, D, D>(1, 2u));
   test(std::extents(1, 2u, 3, 4, 5, 6, 7, 8, 9),
        std::extents<std::size_t, D, D, D, D, D, D, D, D, D>(1, 2u, 3, 4, 5, 6, 7, 8, 9));
+  test(std::extents(S{1}, S{2}), std::extents<std::size_t, D, D>(1, 2u));
```

Comment:
By the way why are we using the unsigned integer suffix literal here?

Reply:
cahnged